### PR TITLE
Added some applications of numpy

### PR DESCRIPTION
**Pull request by Dsantra92**
[Colab link](https://colab.research.google.com/github/S4DS-IEM/Tutorials-Repo/blob/numpy-app/Numpy/Applications.ipynb)

### CODE REVIEW

**Comment by Dsantra92** [comment]
There might some techincal and grammatical mistakes in this notebook. Please review cauitiously 🏳️.

File: Numpy/Applications.ipynb [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+        "Calculate and plot the displacement of an damped pendulum with respect to time."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3YvqIr_1MrSw"
+      },
+      "source": [
+        "A damped pendulum unlike an ideal pendulum stops with time, it's displacement from the center decreaes with time until it comes to zero. So here the maximum displacement is also a function of time which decrases with increasing time. We choose the maximum displacement($\\theta_{max}$) to be a exponentially decaying function of $t$. The displacement function $\\theta(t)$ can therefore be written as:\n",
```
Comment (sdk2k01) [inline Numpy/Applications.ipynb:204]: spelling mistake at line no. 204 of codespace ['decreases' mispelled twice]

File: Numpy/Applications.ipynb [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8nnMMAN6OMbM"
+      },
+      "source": [
+        "<details>\n",
+        "<summary>\n",
+        "Hint 👇\n",
+        "</summary>\n",
+        "The problem is very similar to the previous problem, except you have a exponentially decreasing amplitude. Instead of multiplying a scalar amplitude value like the previous mulitply a array elementwise instead. \n",
```
Comment (sdk2k01) [inline Numpy/Applications.ipynb:225]: 'an' exponentially decreasing amplitude instead of 'a'

File: Numpy/Applications.ipynb [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aNHO6dj7LZMZ"
+      },
+      "source": [
+        "## 😱 Problem 2: A more real life pendulum \n",
+        "Calculate and plot the displacement of an damped pendulum with respect to time."
```
Comment (sdk2k01) [inline Numpy/Applications.ipynb:195]: 'a damped pendulum' instead of 'an damped pendulum'

**Review by sdk2k01 — COMMENTED** [review]
LGTM....marvellous work on the code part (as always)....just would recommend checking the grammatical error analysis.

**Review by nirmalya8 — APPROVED** [review]

**Review by nirmalya8 — COMMENTED** [review]
okay

**Pull request merged**